demo/chatbot_simple/run.py

```diff
@@ -3,7 +3,7 @@
 import time
 
 with gr.Blocks() as demo:
-    chatbot = gr.Chatbot()
+    chatbot = gr.Chatbot(show_chat_copy_button=True)
```

Comment:
I'd revert this change, this demo is intentionally designed to be as simple as possible. Furthermore, this parameter won't actually have any effect since there's no way to populate this chatbot with messages.
```suggestion
    chatbot = gr.Chatbot()
```